snow/consensus/snowball/binary_snowball.go

```diff
@@ -5,8 +5,6 @@ package snowball
 
 import "fmt"
 
-var _ BinarySnowball = (*binarySnowball)(nil)
```

Comment:
switch from removing to ensure it implements `BinarySnow`. Same for similar changes below.